private/eseses/sioCron.sh

```diff
@@ -1,5 +1,12 @@
 #crontab -e
 #0 0 * * * ~/ESESES/private/eseses/cron.sh
+source ~/.bashrc
 cd ~/ESESES/private/eseses
 python metadata.py
 python timeseries.py
+# These are (?) run from within server.js
```

Comment:
Yes, these are executed by the web app.

Reply:
I stuck this in the cron at some point because I found that the position and velocity json files were quite (months) outdated and I thought maybe it was supposed to run out of cron.  I now think this was probably tangled up with the web service changing to port 8080.  Seems to update as expected with the app now.  I'll keep my eye on it.